.env.example

```diff
@@ -0,0 +1,8 @@
+# TMALL API #
+TMALL_API_SAND_HOST_URL='http://gw.api.tbsandbox.com'
```

Comment:
TMALL_API_SAND_HOST_URL change to TMALL_API_SANDBOX_HOST_URL

Reply:
change TMALL_API tp TAOBAO

Reply:
deleted .env